### DISCUSSION TITLE
Disable popups with cmd tab completion

### DISCUSSION THREAD

**Discussion by bojohnson5** [folke/noice.nvim#441] [Q&A] [answered]
I've recently started using LazyVim and am getting it configured to my liking. I've changed `cmdline` `view` option to the classic experience, but when doing tab-completion I still get the popup in the middle of the screen.
![Screen Shot 2023-04-25 at 11 54 26 AM](https://user-images.githubusercontent.com/20647190/234336668-fd8fed6e-c6cd-4326-a38b-a991e1644e34.png)
I'm playing around with different settings but not sure the one to change to disable that popup only for the `cmdline` tab completion. I've disabled `popupmenu` but then I lose lsp popups. Is there a setting I can change to get this behavior I'm looking for?

**Comment by folke** [comment, marked as answer]
You need to disable the `command_palette` preset

**Reply by bojohnson5** [reply]
Thanks for the quick response, cheers!
